content/en/docs/Tasks/creating-operator-manifests.md

```diff
@@ -190,7 +190,7 @@ spec:
       kind: Memcached
 ```
 
-Dependency resolution and ownership is discussed more in depth in the [Concepts section](/docs/Concepts/olm-architecture/dependency-resolution).
+Dependency resolution and ownership is discussed more in depth in the [Concepts section](../../concepts/olm-architecture/dependency-resolution/).
```

Comment:
Can we use the absolute path instead of the relative path instead? 

```suggestion
Dependency resolution and ownership is discussed more in depth in the [Concepts section](/docs/concepts/olm-architecture/dependency-resolution/).
```
I think the mistake here was `Concepts` instead of `concepts`